### DISCUSSION TITLE
Floors!

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#2114] [Announcements]
Hi @home-assistant/language-leaders 

In the 2024.4 release of Home Assistant, we will be adding [floors](https://rc.home-assistant.io/blog/2024/03/27/release-20244/#floors-of-your-home), which group together areas.

In a [recent PR](https://github.com/home-assistant/intents/pull/2111), I added a few English sentences for turning the lights on and off on a floor. The new `{floor}` slot list is now available for use in sentences, and a `floor` attribute can be added to test fixtures:

```yaml
areas:
  - name: "Kitchen"
    id: "kitchen"
    floor: "First Floor"
```

For now, only `HassTurnOn` and `HassTurnOff` intents support floors. We will consider other intents after seeing more how floors are used in the community.

Like turning on/off all lights, etc. in the entire house, please make sure floor sentences are unique and very clear. We want the user to know for sure that multiple areas are going to be affected.

**Comment by mib1185** [comment]
in progress for DE (#2121 thx @steffenrapp )
